helm-chart-validation/action.yml

```
@@ -37,10 +37,10 @@ runs:
       uses: greenbone/actions/set-github-user@v3
 
     - name: Clone Additional Repository
-      if: inputs.repository-url && inputs.repository-branch
+      if: inputs.repository-name && inputs.repository-branch
       uses: actions/checkout@v2
```

You need to clone both the "working" repository and the `greenbone/product-helm-chart` (into separate directories) by using that action twice!

```suggestion
      uses: actions/checkout@v4
```